src/views/Admin/Index.vue

```diff
@@ -66,104 +66,141 @@
+              <path class="opacity-75" fill="currentColor" d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"></path>
+            </svg>
+          </div>
+            <div v-else class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-5 text-gray-900">
+            
+              <!-- project count -->
+              <div class="order-1 lg:order-2 bg-white shadow-md p-3 text-center flex flex-col animate-fade-in-down" style="animation-delay: 0.1s">
+                <h2 class="text-3xl mb-3 text-center font-semibold">Total Projects</h2>
+                <div
+                  class="text-8xl pb-4 font-semibold flex-1 flex items-center justify-center"
+                >
+                  {{ data.totalProjects }}
                 </div>
-            </div>
-            <!-- END Body -->
-        
-            <!-- Action Link -->
-            <a href="#" class="block p-3 font-medium text-sm text-center bg-gray-50 hover:bg-gray-100 hover:bg-opacity-75 active:bg-gray-50 text-indigo-600 hover:text-indigo-700 dark:text-white dark:bg-gray-900/50 dark:hover:text-indigo-300 dark:hover:bg-gray-900/75 dark:active:bg-gray-900/50">
-                View Projects 
-            </a>
-            <!-- END Action Link -->
-            </div>
-            <!-- END Card -->
-        
-            <!-- Card -->
-            <div class="flex flex-col rounded-lg shadow-sm bg-white overflow-hidden dark:text-gray-100 dark:bg-gray-800">
-            <!-- Body -->
-            <div class="p-5 grow flex justify-between items-center">
-                <dl class="space-y-1">
-                <dt class="text-2xl font-bold">
-                    5
-                </dt>
-                <dd class="uppercase font-semibold text-sm text-gray-500 tracking-wider dark:text-gray-400">
-                    Blog Posts
-                </dd>
-                </dl>
-                <div class="font-semibold inline-flex px-2 py-1 leading-4 items-center space-x-1 text-sm rounded-full text-indigo-800 bg-indigo-200">
-                <svg class="hi-solid hi-arrow-circle-down inline-block w-4 h-4" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zm1-11a1 1 0 10-2 0v3.586L7.707 9.293a1 1 0 00-1.414 1.414l3 3a1 1 0 001.414 0l3-3a1 1 0 00-1.414-1.414L11 10.586V7z" clip-rule="evenodd"/></svg>
-                <span>6.9%</span>
+              </div>
+              <!-- /project count -->
+            
+              <!-- blog post -->
+              <div class="order-2 lg:order-2 row-span-2 bg-white shadow-md p-3 text-center flex flex-col animate-fade-in-down" style="animation-delay: 0.2s">
+                <h2 class="text-3xl mb-3 text-center font-semibold">Blog Post</h2>
+                <div v-if="data.latestPosts" class="text-left">
```

Comment:
`    <div
                class="text-8xl pb-4  font-semibold flex-1 flex items-center justify-center"
              >
                {{ data.totalPosts }}
              </div>
                <div v-if="data.latestPosts && data.latestPosts.length > 0" class="text-left">
                  <a
                  v-for="post in data.latestPosts"
                  :key="post.id"
                  :href="post.url"
                  class="block p-2 hover:bg-gray-100/90"
                  target="_blank"
                  >
                  <div class="font-semibold">{{ post.title }}</div>
                  <small>
                    Answer Made at:
                    <i class="font-semibold">{{ formatDate(post.created_at) }}</i>
                  </small>
                </a>
                </div>
                <div v-else class="text-gray-600 text-center py-16">
                  You have not made any post yet
                </div>`

Reply:
To display the latest post details in the dashboard, you need to iterate over the data.latestPosts collection and access the properties of each individual post.